src/PanelGroup.js

```diff
@@ -379,13 +380,6 @@ var Panel = React.createClass({
 
 var Divider = React.createClass({
 
-  getDefaultProps: function() {
-    return {
-      dividerWidth: 1,
-      handleBleed: 4,
-    };
-  },
-
```

Comment:
I essentially moved them to `PanelGroup`'s default props. Since `Divider` is an internal component that is always instantiated by `PanelGroup`, which explicitly specifies all its properties, I figured the `Divider`'s default props are redundant and I didn't want leave the effectively unused defaults hanging around confusing potential future contributors :)